### ISSUE TITLE
TF2.0 : NonMaxSuppressionV2GPUOp segmentation fault

### ISSUE THREAD

**Issue by wazeerzulfikar** [tensorflow/tensorflow#32261]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): TF2.0.0-rc0
- Python version:
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: CUDA 10.0
- GPU model and memory: GeForce RTX 2080 Ti

**Describe the current behavior**
TF2.0 has a Non Max Suppression V2 op gpu implementation which seg faults when it is run.
Due to this [TF object detection model zoo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md) saved models cannot be run in TF2.0.

**Code to reproduce the issue**
```
import tensorflow as tf
from tensorflow.python.ops import gen_image_ops

with tf.device('/device:GPU:0'):
    boxes = tf.random_uniform((10,4), dtype=tf.float32)
    scores = tf.constant([1]*10, tf.float32)
    max_output_size = tf.constant(1)
    iou_threshold = tf.constant(0.5)

    nms = gen_image_ops.non_max_suppression_v2(boxes, scores, max_output_size, iou_threshold)
    print(nms)
```

**gdb output** 
```
Thread 1 "python3" received signal SIGSEGV, Segmentation fault.                                                                                                                          
0x00007fff4372b741 in tensorflow::NonMaxSuppressionV2GPUOp::Compute(tensorflow::OpKernelContext*) ()                                                                                     
   from /usr/local/lib/python3.6/dist-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so
```

**Comment by ppwwyyxx** [comment]
cc @samikama

**Comment by ymodak** [comment]
I think this is fixed with tf 2.0 nightly version '2.0.0-dev20190913'. Can you please confirm? Thanks!
```python
import tensorflow as tf
from tensorflow.python.ops import gen_image_ops

with tf.device('/device:GPU:0'):
    boxes = tf.random.uniform((10,4), dtype=tf.float32)
    scores = tf.constant([1]*10, tf.float32)
    max_output_size = tf.constant(1)
    iou_threshold = tf.constant(0.5)

    nms = gen_image_ops.non_max_suppression_v2(boxes, scores, max_output_size, iou_threshold)
    print(nms)
```
Output:
```python
tf.Tensor([0], shape=(1,), dtype=int32)
```

**Comment by Icaro-Lima** [comment]
> I think this is fixed with tf 2.0 nightly version '2.0.0-dev20190913'. Can you please confirm? Thanks!
> 
> ```python
> import tensorflow as tf
> from tensorflow.python.ops import gen_image_ops
> 
> with tf.device('/device:GPU:0'):
>     boxes = tf.random.uniform((10,4), dtype=tf.float32)
>     scores = tf.constant([1]*10, tf.float32)
>     max_output_size = tf.constant(1)
>     iou_threshold = tf.constant(0.5)
> 
>     nms = gen_image_ops.non_max_suppression_v2(boxes, scores, max_output_size, iou_threshold)
>     print(nms)
> ```
> 
> Output:
> 
> ```python
> tf.Tensor([0], shape=(1,), dtype=int32)
> ```

Thanks! It works here!

**Comment by ynjiun** [comment]
or install newer version > 2.0 such as 
`pip install tensorflow-gpu==2.1.0`
and it works for me
